docs/src/installation.md

```diff
@@ -107,6 +107,13 @@ brew install uutils-coreutils
 port install coreutils-uutils
 ```
 
+## FreeBSD
+[![FreeBSD port](https://repology.org/badge/version-for-repo/freebsd/uutils-coreutils.svg)](https://repology.org/project/uutils-coreutils/versions)
+
+```sh
+pkg install uutils
```

Comment:
I wish the package was named differently ... uutils is the org name, not the project ...
do you know if it could be renamed?

Reply:
Well, I am not convinced by the fact that we should follow phoronix's article to name a package.

Could you please rename it?
with the exception of gentoo (which I will ping too), they all have coreutils in the name:
https://uutils.github.io/user/installation.html

Reply:
@pkubaj I created https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=269745 to make sure it isn't forgotten